monitor/monitor_test.go

```diff
@@ -17,6 +19,24 @@ const (
 	logKey = "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAElgyN7ptarCAX5krBwDwjhHM+b0xJjCKke+Dfr3GWSbLm3eO7muXRo8FDDdpdiRpnG4NJT0bdzq5YEer4C2eZ+g=="
 )
 
+func TestMakeDBPermissions(t *testing.T) {
+	f, err := ioutil.TempFile("", "")
+	if err != nil {
+		t.Fatal(err)
+	}
+	err = os.Chmod(f.Name(), 0777)
+	if err != nil {
```

Comment:
nit: you could collapse L28 and L27 into one:
```suggestion
	if err := os.Chmod(f.Name(), 0777); err != nil {
```